Apromore-Core-Components/Apromore-BPMNEditor/src/main/resources/bpmneditor/editor/bpmnio/bpmn-modeler.development.js

```diff
@@ -18980,7 +18981,11 @@ var getBusinessObject = __webpack_require__(1).getBusinessObject,
 
 module.exports = function(element, bpmnFactory, elementRegistry, translate) {
 
-  var tasks = getFlowElementsByType(element, 'bpmn:Task');
+  var taskEl = 'bpmn:Task';
+  if (element.type.endsWith('Task')) {
+    taskEl = element.type;
+  }
+  var tasks = getFlowElementsByType(element, taskEl);
```

Comment:
I'm suspicious about the correctness of this because not every BPMN element named with a "Task" suffix is a subtype of BPMN Task.  ChoreographyTask and GlobalTask are examples.  GlobalTasks are never FlowElements so they're probably safe in this particular case, but ChoreographyTasks are FlowElements so they'd fool this logic.

Reply:
Yeah, I need some guidance for this. I probably need to use type checking is()

Reply:
Can we review on "QBP simulator repo" which is the source repo instead of "bpmn-modeler.development.js" which is the bundle file? It's impossible to review on this huge file.

Reply:
https://github.com/apromore/QBPSimulator/pull/28